views/_partials/jumbotron.pug

```diff
@@ -2,23 +2,21 @@
     h1.display-5 BootstrapCDN
 
     p.lead
-        | The recommended 
-        a(href='https://www.stackpath.com/products/cdn/?utm_campaign=Partner%20Display&utm_source=Partner%20Display&utm_medium=bootstrapcdn', rel='noopener', target='_blank') CDN
+        | The recommended CDN
```

Comment:
Not sure if you want to link to jsdevilr here too or maybe https://en.wikipedia.org/wiki/Content_delivery_network? It's nitpicking, so it's your call :)

Reply:
Yeah maybe in the future. For now let's just do a cleanup :)